packages/core/logger/src/index.js

```diff
@@ -33,137 +32,106 @@ export class Logger {
       }));
     };
     this.logFile = options.logFile;
-    this.context = options.context;
-  }
-
-  /**
-   * Parses the logFile, returning an array of JSON objects containing the
-   * log messages.
-   * @param {Number} limit specifies how many log messages to return from the
-   *  end of the log file
-   * @returns {Array} array containing
-   *  - msg: the log message
-   *  - logLevel: log level (ie 'info', 'debug')
-   *  - name: process name (always "embark")
-   *  - timestamp: timestamp of log message (milliseconds since 1/1/1970)
-   */
-  parseLogFile(limit) {
-    let matches;
-    let logs = [];
-    const logFile = fs.readFileSync(this.logFile, 'utf8');
-    while ((matches = LOG_REGEX.exec(logFile)) !== null) {
-     // This is necessary to avoid infinite loops with zero-width matches
-     if (matches.index === LOG_REGEX.lastIndex) {
-       LOG_REGEX.lastIndex++;
-     }
-
-     if (matches && matches.length) {
-       logs.push({
-         msg: [matches[3]],
-         logLevel: matches[2],
-         name: 'embark',
-         timestamp: date.parse(matches[1], DATE_FORMAT).getTime()
-       });
-     }
-    }
 
-    // if 'limit' is specified, get log lines from the end of the log file
-    if(limit && limit > 0 && logs.length > limit){
-      logs.slice(limit * -1);
-    }
-    return logs;
+    this.writeToFile = async.cargo((tasks, callback) => {
+      if (!this.logFile) {
+        return callback();
+      }
+      let logs = '';
+      let origin = "[" + ((new Error().stack).split("at ")[3]).trim() + "]";
+      tasks.forEach(task => {
+        logs += `[${date.format(new Date(), DATE_FORMAT)}] ${task.prefix} ${task.args}\n`;
```

Comment:
The `\n` at the end seems correct. Looking at the generated logs without it, it concatenates multiple messages in an unreadable way otherwise. 

I'm updating this part though. Not making use of `join()` and do simple string concatenation instead.